Server from README doesn't work

I've tried to run server using the example from README:
```eo
# Create TCP server and echo

+alias org.eolang.net.socket

seq > @
  write.
    as-output. > output
      accept. > conn
        listen.
          socket
            "localhost"
            8080
    conn.as-input.read
  output.flush
```

I get the next exception:
```java
Caused by: java.io.FileNotFoundException: https://raw.githubusercontent.com/objectionary/home/67da11ac07940770acdea1d86b862755a6af3ca5/objects/org/eolang/net/socket.eo
```

Full exception log:
[server.log](https://github.com/objectionary/eo-net/files/10130464/server.log)

--- a/README.md
+++ b/README.md
@@ -12,14 +12,15 @@ This is how to connect to a TCP server and send 88:
 ```
 +alias org.eolang.net.socket
 
-seq > @
-  as-output. > output
-    connect.
-      socket
-        "localhost"
-        8080
-  output.write 88
-  output.flush
+[] > main
+  seq > @
+    as-output. > output
+      connect.
+        socket
+          "localhost"
+          8080
+    output.write 88
+    output.flush
 ```
 
 Server:
@@ -28,16 +29,17 @@ Server:
 
 +alias org.eolang.net.socket
 
-seq > @
-  write.
-    as-output. > output
-      accept. > conn
-        listen.
-          socket
-            "localhost"
-            8080
-    conn.as-input.read
-  output.flush
+[] > main
+  seq > @
+    write.
+      as-output. > output
+        accept. > conn
+          listen.
+            socket
+              "localhost"
+              8080
+      conn.as-input.read
+    output.flush
 ```
 
 ## How to Contribute

